chall02/jsalmi.c

```diff
@@ -0,0 +1,35 @@
+#include <stdlib.h>
+
+char puthex(int nb)
+{
+	char c;
+
+	if (nb >= 10)
+	{
+		nb == 10 ? c = 'a' : 0;
+		nb == 11 ? c = 'b' : 0;
+		nb == 12 ? c = 'c' : 0;
+		nb == 13 ? c = 'd' : 0;
+		nb == 14 ? c = 'e' : 0;
+		nb == 15 ? c = 'f' : 0;
+	}
```

Comment:
could have been done another way, it's not an if-forest but it's not really elegant. Plus, It will check all those conditions, not really well optimized.